Makefile.jenkins

```diff
@@ -0,0 +1,69 @@
+# This Makefile is a support file to automatize the kiali release process. It is
+# used in a Jenkins Pipeline.
+# It can also be used to run the release process without Jenkins. Read hte
+# RELEASING.adoc file for more information.
+
+UI_GITHUB_URI ?= git@github.com:kiali/kiali-ui.git
+UI_FORK_URI ?= $(shell git config --get remote.origin.url)
+UI_PULL_URI ?= https://api.github.com/repos/kiali/kiali-ui/pulls
+
+UI_VERSION ?= $(shell jq -r '.version' package.json)
+UI_BUMPED_VERSION ?= $(shell semver bump minor $(UI_VERSION))
+UI_VERSION_BRANCH ?= $(shell jq -r '.version' package.json | sed 's/\.0$$//')
+
+BUILD_TAG ?= prepare-next-version
+BUMP_BRANCH_ID ?= $(BUILD_TAG)
+
+NPM_DRY_RUN ?= n
+
+.PHONY: all ui-bump-minor-version ui-build ui-test ui-npm-publish
+.PHONY: ui-push-version-tag ui-create-pr-next-version release
+
+all:
+	$(error You must explicitly specify a target)
+
+ui-bump-minor-version:
+	jq -r '.version |= "$(UI_BUMPED_VERSION)"' package.json > package.json.bumped
+	mv package.json.bumped package.json
+
+ui-build:
+	rm -fr build/*
```

Comment:
`-fr` !? -Nothing requested here, only that i'm accustomed to write/hear `rm -rf`-

Reply:
I don't know if I always use that order or if I use both indistinctly :)